testing/test_config.py

```diff
@@ -824,8 +839,12 @@ def test_invocation_params_args(self, _sys_snapshot) -> None:
         assert config.option.verbose == 4
         assert config.option.capture == "no"
 
-    def test_inifilename(self, tmpdir):
-        tmpdir.join("foo/bar.ini").ensure().write(
+    def test_inifilename(self, tmp_path: Path) -> None:
+        d1 = tmp_path.joinpath("foo")
+        d1.mkdir()
+        p1 = d1.joinpath("bar.ini")
+        p1.touch()
+        p1.write_text(
```

Comment:
is there a shortcut to create file with path at once, instead of `"foo" mkdir() -> "bar.ini" touch()`? I was getting file not found error for `tmp_path.joinpath("foo/bar.ini").touch()`.